webapp/packages/core-utils/src/OrderedMap.test.ts

```diff
@@ -0,0 +1,143 @@
+import { OrderedMap } from './OrderedMap';
+
+describe('OrderedMap', () => {
+  it('should add and get items', () => {
+    const map = new OrderedMap<number, string>();
+    map.add(1, 'one');
+    map.add(2, 'two');
+    map.add(3, 'three');
+
+    expect(map.get(1)).toBe('one');
+    expect(map.get(2)).toBe('two');
+    expect(map.get(3)).toBe('three');
+  });
```

Comment:
please add negative checks like `get not exist item returns undefined`, `has returns false for not exist element`